lib/flag_shih_tzu.rb

```diff
@@ -23,19 +24,25 @@ module ClassMethods
     def has_flags(*args)
       flag_hash, opts = parse_flag_options(*args)
       opts = {
-        :named_scopes => true,
-        :column => DEFAULT_COLUMN_NAME,
-        :flag_query_mode => :in_list, # or :bit_operator
-        :strict => false,
-        :check_for_column => true
+          :named_scopes => true,
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.